system/application/views/widgets/mediaelement/jquery.mediaelement.js

```diff
@@ -5538,6 +5538,8 @@ function YouTubeGetID(url){
 
 			this.parentView.layoutMediaObject();
 			this.parentView.removeLoadingMessage();
+      // make sure mirador doesn't overflow its bounds
+      $('.mirador-viewer').css('max-height', $('.mediaContainer').css('max-height'));
```

Comment:
Is it possible for there to be more than one Mirador viewer on the page at a time? If so, it might be worth passing  the context for the jQuery selectors and/or referencing the element directly if possible. Not sure if this makes sense, but maybe something like this?

```javascript
$('.mirador-viewer', this.mediaObject).css('max-height', $(this.parentView.mediaContainer).css('max-height'));
```

Also, looks like this file is indented with tabs instead of spaces.

Reply:
Great suggestion, and change made. I recommitted the code with tabs and it results in the same spacing. Not sure why it looks good for me, but it turns out like this on github.